`clever addon delete` need always `--orga` ?

I tried to delete a postgresql addon with `clever addon delete` but I need to force the `--orga`

```
export DATABASE_ID=$(clever curl $CURL_APP_URL/addons | jq '.[] | select(.provider.id == "postgresql-addon") | .id' -r)
clever addon delete $DATABASE_ID --verbose --yes
[DEBUG] Load configuration from environment variables
[DEBUG] Load configuration from /home/apasquier/.config/clever-cloud/clever-tools.json
[DEBUG] DELETE https://api.clever-cloud.com/v2/self/addons/addon_909f7994-30b3-403e-9c11-16d9bf4be363 ? undefined
```

Only `/self`  is used.

The `$DATABASE_ID` is a valid one because `clever addon delete $DATABASE_ID --org=$ORGA_ID --yes` works

I tried on two add-ons here in a personal org, with name and add-on ID, it was ok: 

```
clever addon delete mbTest
Deleting the addon can't be undone, are you sure? y
Addon mbTest successfully deleted

clever addon delete addon_id
Deleting the addon can't be undone, are you sure? y
Addon addon_id successfully deleted
```

In another org, it was not OK, it was not found:

```
clever addon delete addon_id
Deleting the addon can't be undone, are you sure? y
[ERROR] This addon does not exist.

clever addon delete kvTest
[ERROR] Addon not found
```

With the `-o/--org` it was ok

--- a/src/commands/addon.js
+++ b/src/commands/addon.js
@@ -176,7 +176,17 @@ async function deleteAddon (params) {
   const { yes: skipConfirmation, org: orgaIdOrName } = params.options;
   const [addon] = params.args;
 
-  const ownerId = await Organisation.getId(orgaIdOrName);
+  let ownerId = await Organisation.getId(orgaIdOrName);
+
+  if (ownerId == null && addon.addon_id != null) {
+    ownerId = await Addon.findOwnerId(ownerId, addon.addon_id);
+  }
+
+  if (ownerId == null && addon.addon_name != null) {
+    const foundAddon = await Addon.findByName(addon.addon_name);
+    ownerId = foundAddon.orgaId;
+  }
+
   await Addon.delete(ownerId, addon, skipConfirmation);
 
   Logger.println(`Addon ${addon.addon_id || addon.addon_name} successfully deleted`);

--- a/src/models/addon.js
+++ b/src/models/addon.js
@@ -243,6 +243,22 @@ async function findById (addonId) {
   throw new Error(`Could not find add-on with ID: ${addonId}`);
 }
 
+async function findByName (addonName) {
+  const { user, organisations } = await getSummary({}).then(sendToApi);
+  for (const orga of [user, ...organisations]) {
+    for (const simpleAddon of orga.addons) {
+      if (simpleAddon.name === addonName) {
+        const addon = await getAddon({ id: orga.id, addonId: simpleAddon.id }).then(sendToApi);
+        return {
+          ...addon,
+          orgaId: orga.id,
+        };
+      }
+    }
+  }
+  throw new Error(`Could not find add-on with name ${addonName}`);
+}
+
 async function findOwnerId (org, addonId) {
 
   if (org != null && org.orga_id != null) {
@@ -305,6 +321,7 @@ module.exports = {
   delete: deleteAddon,
   findById,
   findOwnerId,
+  findByName,
   getProvider,
   getProviderInfos,
   link,

